src/features/movies/MovieList/movieListSaga.js

```diff
@@ -26,6 +29,9 @@ function* fetchMovieListHandler() {
     catch (error) {
         yield call(alert, "Error fetching popular movies!");
     }
+    finally {
+        yield put(setLoading(false));
```

Comment:
like in previous comment, I think we can omit this part if we update setError and setMovieList with also updating `status` property of the state.

Reply:
Noted! I will correct this 😉